lib/rubocop/cop/style/multiline_when_then.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks uses of the `then` keyword
+      # in multi-line when statements.
+      #
+      # @example
+      #   # bad
+      #   case str
+      #   when 'rubocop' then
+      #     puts 'rubocop'
+      #   else
+      #     puts 'not rubocop'
+      #   end
+      #
+      #   # good
+      #   case str
+      #   when 'rubocop'
+      #     puts 'rubocop'
+      #   else
+      #     puts 'not rubocop'
+      #   end
+      #
+      #   # good
+      #   case str
+      #   when 'rubocop' then puts 'rubocop'
+      #   else
+      #     puts 'not rubocop'
+      #   end
+      #
```

Comment:
How about the following example code based on https://github.com/rubocop-hq/ruby-style-guide/pull/752?

```ruby
# bad
case foo
when bar then
end

# good
case foo
when bar
end

# good
case foo
when bar then do_something
end
```

Reply:
Right.